.gitignore

```diff
@@ -13,6 +13,7 @@ WindowsStoreApp/
 UnityGenerated/
 project.json
 project.lock.json
+ProjectSettings/ProjectVersion.txt
```

Comment:
Woah, why? Doesn't Unity use this to determine which version the assets are are made with, then uses that info to migrate to new versions? This seems dangerous and could result in assets being updated incorrectly.

Reply:
I can revert this.

Backwards compatibility for Unity versions is more dangerous than forwards compatibility.  Usually Unity handles updating any meta files without much worry.

Reply:
I suppose we could make the ProjectVersion.txt reflect the minimum editor version compatibility (which is currently 5.6)

Reply:
Our ProjectVersion.txt should be the latest released and tested Unity version.

Reply:
If the recommended way to get the Toolkit is via our released UnityPackages, that shouldn't be a problem. We can require a version of Unity for the Toolkit project that is different from the version used in an app.

Reply:
I think targeting the min version is probably better if you want to avoid asset corruption.

Reply:
The Toolkit's version is 2017.2. Nothing before that supports full mixed reality development. If an earlier version is needed, we have several releases for that.